setup.py

```diff
@@ -14,7 +14,7 @@
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
     packages=find_packages(),
-    author="Greg Caporaso",
+    author="Greg Caporaso and Michael Robeson",
```

Comment:
👍 --- maybe we can think about extending plugin registration to support per-method authors some day in the future. Either way, not sure if we need to also update the `author_email` below - @gregcaporaso & @ebolyen, thoughts?